projects/go-lib/src/lib/components/go-table/go-table.component.ts

```diff
@@ -66,6 +66,7 @@ export class GoTableComponent implements OnInit, OnChanges, AfterViewInit {
 
   @ContentChildren(GoTableColumnComponent) columns: QueryList<GoTableColumnComponent>;
   @ContentChild('goTableDetails', { static: false }) details: TemplateRef<any>;
+  @ContentChild('goTableTitle', { static: false }) tableTitleTemplate: TemplateRef<any>;
```

Comment:
I feel like the title should be static, but I could maybe see the case where an icon might change or something